README.md

```diff
@@ -108,8 +108,8 @@ Contributions are always welcome! Please read the [contribution guidelines](http
 
 ## Contributors
 
-- [Jonathan L. Robinson](https://www.chalmers.se/en/Staff/Pages/jonrob.aspx), National Bioinformatics Infrastructure Sweden (NBIS), Science for Life Laboratory (SciLifeLab), Chalmers University of Technology, Gothenburg Sweden
-- [Pınar Kocabaş](https://www.chalmers.se/en/staff/Pages/kocabas.aspx), Chalmers University of Technology, Gothenburg Sweden
-- [Pierre-Etienne Cholley](https://www.chalmers.se/en/staff/Pages/cholley.aspx), Chalmers University of Technology, Gothenburg Sweden
-- [Avlant Nilsson](https://www.chalmers.se/en/staff/Pages/avlant-nilsson.aspx), Chalmers University of Technology, Gothenburg Sweden
-- [Hao Wang](https://www.chalmers.se/en/staff/Pages/hao-wang.aspx), Chalmers University of Technology, Gothenburg Sweden
+- [Jonathan L. Robinson](https://bii.dk/team/jonathan-robinson/), The BioInnovation Institute (BII), Copenhagen, Denmark
+- [Pınar Kocabaş](https://research.chalmers.se/en/person/kocabas), Chalmers University of Technology, Gothenburg, Sweden
+- [Pierre-Etienne Cholley](https://www.chalmers.se/en/staff/Pages/cholley.aspx), Chalmers University of Technology, Gothenburg, Sweden
+- [Avlant Nilsson](https://www.chalmers.se/en/staff/Pages/avlant-nilsson.aspx), Chalmers University of Technology, Gothenburg, Sweden
+- [Hao Wang](https://www.chalmers.se/en/staff/Pages/hao-wang.aspx), Chalmers University of Technology, Gothenburg, Sweden
```

Comment:
I hope it's okay to bring this up again about maintaining a _Contributors_ section https://github.com/SysBioChalmers/Human-GEM/issues/176#issuecomment-640478034

Reply:
It wasn't clear to me from the linked thread, but are you suggesting to just remove this contributors section from the Readme because it's redundant with information already present on GitHub?

Reply:
>  remove this contributors section from the Readme

Yes that was what I meant. Now the _Contributors_ are even highlighted on the right part of the landing page of the repository. Also, Yeast-GEM has already removed this section.

Reply:
Ah ok, then yes, I completely agree.